.circleci/config.yml

```diff
@@ -39,6 +39,27 @@ jobs:
       - deploy:
           command: make -C .doks/ deploy_to_doks
 
+  validate_deployment_status:
+    docker:
+      - image: codacy/ci-do:0.2.3
+    environment:
+      DOKS_CLUSTER_NAME: codacy-doks-cluster-nightly
+    working_directory: ~/workdir/
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: "Setup DO Credentials"
+          command: doctl auth init -t $DO_TOKEN &>/dev/null
+      - run:
+          name: "Validate deployment status"
+          command: bash ./.circleci/validateDeploymentStatus.sh
```

Comment:
you don't need the bash here.. you've put a shebang on the script

Reply:
I would also like it more to have this inline. But if you like it more this way its ok for me too